server/routes/alumni.js

```diff
@@ -149,6 +150,11 @@ router.post('/', async (req, res, next) => {
         );
         await alumni_instance.save();
         await user_instance.save();
+        const news_instance = new newsSchema({
+            event: 'New Alumni',
+            alumni: [alumni_instance._id],
+        })
+        await news_instance.save();
```

Comment:
Template for adding a news item - most fields have defaults or aren't required, so the only necessary field is the event.
This means we can relatively cleanly make news items for multi-alumni/student activities!